charts/tractusx-connector/values.yaml

```
@@ -241,9 +241,11 @@ controlplane:
     otel.javaagent.debug=false
   # -- configuration of the [Java Util Logging Facade](https://docs.oracle.com/javase/7/docs/technotes/guides/logging/overview.html)
   logging: |-
-    .level=INFO
+    .level=FINE
```

Shouldn't this be still this

```suggestion
    .level=INFO
```